terrain/voxel_terrain.cpp

```diff
@@ -1018,15 +1018,13 @@ void VoxelTerrain::_process() {
 					continue;
 				}
 
+				collidable_surfaces.push_back((Array*)&data.blocky_surfaces.surfaces[i]);
```

Comment:
No need to cast because it should not need to be a pointer type, as said earlier